requirements.txt

```diff
@@ -1,7 +1,7 @@
 boto3 ~= 1.14
 deprecation ~= 2.1.0
 pynacl ~= 1.4
-pystac ~= 0.5.3
+pystac >= 0.5.3
```

Comment:
The latest version is not compatible with python 3.8, so we need some kind of range here.

Reply:
(I originally had this pinned to the latest pystac version but realized it wouldn't work with python 3.8)